lib/fika/parser.ex

```diff
@@ -72,6 +72,14 @@ defmodule Fika.Parser do
     |> label("boolean")
     |> Helper.to_ast(:boolean)
 
+  atom =
+    ":"
+    |> string()
+    |> ignore()
+    |> concat(identifier)
+    |> label("literal")
```

Comment:
I think you forgot to change this "literal" to "atom" :)

Reply:
Might have gotten lost while removing the stray formatting changes!